aait/backend/group-2A/Application/Features/User/Request/Commands/CreateUserCommand.cs

```diff
@@ -5,12 +5,13 @@
 using System.Threading.Tasks;
 using Application.DTO.Post;
 using Application.DTO.UserDTO;
+using Application.Model;
 using Domain;
 using MediatR;
 
 namespace Application.Features.User.Request.Commands
 {
-    public class CreateUserCommand : IRequest<int>
+    public class CreateUserCommand : IRequest<AuthResponse>
```

Comment:
You can have registration response like :  public class RegistrationResponse
    {
        public string UserId { get; set; }
    }